### PR TITLE
RDISCROWD-3501 Create Number of Priority 1 Tasks Info Field

### PR DESCRIPTION
**Pull request by XiChenn**
*Issue number of the reported bug or feature request: RDISCROWD-3501*

**Describe your changes**
Add a line in the UI which read a variable in the template. The variable gets the value from the DB. Also made two wording changes as requested.

**Testing performed**
All tests passed locally.

**Related PR**
https://github.com/bloomberg/pybossa-default-theme/pull/291

Number of Priority 1 Tasks

### CODE REVIEW

File: pybossa/cache/helpers.py [hunk 1]
```diff
@@ -235,3 +237,21 @@ def n_locked_tasks(project_id):
     from pybossa.redis_lock import get_active_user_count
 
     return get_active_user_count(project_id, sentinel.master)
+
+
+def n_priority_x_tasks(project_id, priority=1.0, include_gold_task=False):
+    """Return the number of tasks with a given priority for a given project."""
+    if include_gold_task:
+        query = text('''SELECT COUNT(*) AS n_tasks FROM task
+                        WHERE project_id=:project_id AND state !='completed'
```
Comment (dchhabda) [inline pybossa/cache/helpers.py:246]: state can be searched here and at line 250 as 
```
state = 'ongoing'
```
also, docstring can be updated to include ongoing tasks

Comment (XiChenn) [reply pybossa/cache/helpers.py:246]: Thanks Deep. I think "ongoing" is clearer and less error pone than "reversed completed". Will update it. Curious, does technically our system treat not 'completed' as 'ongoing'? If there are some others states like 'error' then the answer would be no.

**Review by dchhabda — APPROVED** [review]

File: pybossa/cache/helpers.py (hunk 1 above)
Comment (dchhabda) [reply pybossa/cache/helpers.py]: thats correct, tasks not completed are marked as ongoing.

**Review by dchhabda — COMMENTED** [review]
As discussed, please submit themes changes and update submodule. Thanks!

**Comment by dchhabda** [comment]
@XiChenn , you can merge theme's pr first and then update this pr pointing to latest theme's sha from master branch.

**Review by dchhabda — DISMISSED** [review]

**Review by dchhabda — APPROVED** [review]

**Pull request merged**